File: Python/mystuff/calculator/calculator.py
"""

calculator.py by AJ Audet

The purpose of this program is to create a calculator using python,

"""

import math, random, sys

# HELP AAAAAAAAA


# general variables
x = input("Enter first int: ")
y = input("Enter second int: ")

switch = input("1. Add, 2. Subtract, 3. Multiply, 4. Divide, 5. Exponential\n")

# switches in python dont exist RAH, its all elif
# free me from this language
if switch == 1:
    sum = x + y
    print("The sum of x and y is: ", + sum)
elif switch == 2:
    sum = x - y
    print("The sum of x and y is: ", + sum)
elif switch == 3:
    product = x * y
    print("The product of x and y is: " + product)
elif switch == 4:
    product = x / y
    print("The product of x and y is: " + product)
elif switch == 5:
    num = math.pow(x, y)
    print("X raised to the power of Y is ", + num)

File: C++/mystuff/calculator/calculator.cpp
/*
    calculator.cpp by AJ Audet

    makin a calculator using standard math functions, so base arithmatic, add, subtract, multiply, divide, modulo, 
        power, root, square, sqrt

*/

#include <iostream>
#include <math.h>
using namespace std;

void add(double x, double y)
{
    result = x + y;
}

void subtract(double x, double y)
{
    reuslt = x - y;
}

void multiply(double x, double y)
{
    reuslt = x * y;
}

void divide(double x, double y)
{
    reuslt = x / y;
}

void square(double x)
{
    reuslt = x * x;
}

void power(double x, double y)
{
    reuslt = pow(x, y);
}

void root(double x, double y)
{
    reuslt = x ^ (1/y);
}

void modulo(double x, double y)
{
    reuslt = x % y;
}

void sqRoot(double x)
{
    reuslt = sqrtl(x);
}

int main()
{
    double x, y;
    int control;

    cout << "1. Add, 2. Subtract, 3. Multiply, 4. Divide, 5. Square, 6. Power of, 7. Root of, 8. Modulo, 9. Square Root : ";
    cin >> control;
 
    switch (control)
    {
    case 1:
        cout << "Enter a number: ";
        cin >> x;
        cout << "Enter a number: ";
        cin >> y;
        add(x, y);
        break;
    case 2:

        break;

    case 3: 

        break;

    case 4:
        
        break;
    case 5:

        break;

    case 6: 

        break;

    case 7:
        
        break;

    case 8:

        break;

    case 9: 

        break;

    default:
        cout << "Please Choosse a Function!" << endl;
        break;
    }
}